test/functional/feature_bip8.py

```diff
@@ -0,0 +1,124 @@
+#!/usr/bin/env python3
+# Copyright (c) 2021 The Bitcoin Core developers
+# Distributed under the MIT software license, see the accompanying
+# file COPYING or http://www.opensource.org/licenses/mit-license.php.
+# Test BIP 8 softforks
+from collections import namedtuple
+from test_framework.test_framework import BitcoinTestFramework
```

Comment:
Separate std library imports from local imports

Reply:
If I need to retouch